manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
+
+jobs:
+- name: bosh
+  templates:
+  - {name: collectd, release: collectd}
+
+properties:
+  collectd:
+    interval: 10
+    config: |
```

Comment:
Discussed - not currently possible because they have different prefixes to workaround another issue and we can't interpolate into multiline strings. We should record it as follow-up work.